sysdig/internal/client/secure/models.go

```diff
@@ -352,3 +352,25 @@ func VulnerabilityExceptionFromJSON(body []byte) *VulnerabilityException {
 
 	return &result
 }
+
+// -------- CloudAccount --------
+
+type CloudAccount struct {
+	AccountID     string `json:"accountId"`
+	Provider      string `json:"provider"`
+	Alias         string `json:"alias"`
+	RoleAvailable bool   `json:"roleAvailable"`
+	ExternalID    string `json:"external_id,omitempty"`
```

Comment:
@nkraemer-sysdig Do we want to make this one `externalId` that comes back from the CloudAccount response?